templates/_base.html

```diff
@@ -36,40 +34,33 @@
   </head>
   <body>
   <div id="wrapper">
-  <nav class="navbar navbar-expand-lg navbar-light bg-gradient-light">
+  <nav class="navbar navbar-expand-lg navbar-light bg-gradient-light d-md-none">
     <div class="d-flex flex-grow-1">
 		<a class="navbar-brand" href="{{ base_url }}index.html">Lean Community
     </a>
     </div>
 
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
-    <div class="collapse navbar-collapse flex-grow-1 text-right" id="navbarSupportedContent">
-      <ul class="navbar-nav ml-auto">
-		{% for menu in menus %}
-        <li class="nav-item dropdown">
-          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-			  {{ menu.title }}
-          </a>
-		  <div class="dropdown-menu {{ menu.class_right }}" aria-labelledby="navbarDropdown">
-			{% for item in menu.items %}
-			{% if item.title %}
-			<a class="dropdown-item" href="{{ item.url | url }}">{{ item.title }}</a>
-			{% else %}
-			<div class="dropdown-divider"></div>
-			{% endif %}
-			{% endfor %}
-          </div>
-        </li>
-		{% endfor %}
-      </ul>
-    </div>
   </nav>
-  <div class="container-sm mt-4">
+  <div class="container-fluid"><div class="row">
+    <nav id="navbar" class="col-md-3 col-xl-2 d-md-block bg-light sidebar collapse navbar navbar-light pb-4">
+      <div class="d-sm-none d-md-block"><a class="navbar-brand" href="{{ base_url }}index.html">Lean Community</a></div>
+      {% for menu in menus %}<div>
+        <h6 class="sidebar-heading px-3 mt-4 mb-1">{{ menu.title }}</h6>
+        <ul class="nav flex-column">{% for item in menu.items %}
+          <li class="nav-item"><a href="{{ item.url | url }}">{{ item.title }}</a></li>
```

Comment:
```suggestion
          <li class="nav-item"><a class="nav-link" href="{{ item.url | url }}">{{ item.title }}</a></li>
```

Reply:
Ok, I've used `d-block` though.  The `nav-link` class messes with the padding.